src/main/java/com/sda/eventapp/service/EventService.java

```diff
@@ -139,8 +139,8 @@ public EventView findEventViewById(Long id) {
         return mapper.toEventView(this.findById(id));
     }
 
-    public List<EventView> findAllEventViews() {
-        return mapper.toEventViewList(repository.findAll());
+    public List<EventView> findAllEventViewsFuture() {
```

Comment:
I'd prefer naming of this class to be like `findAllFutureEventViews`. If you guys have other opinion let me know so we can resolve it.